addon/resolver.js

```diff
@@ -105,4 +118,17 @@ export default EmberResolver.extend({
 
     return false;
   },
+
+  /**
+    Checks `type` need use prefix received through a {{#crossLink "DeviceService"}}{{/crossLink}} to lookup class this `type` or not.
```

Comment:
Checks that 'type' needs to be resolved with prefixes received through a {{#crossLink "DeviceService"}}device service{{/crossLink}}